presto-hive/src/main/java/io/prestosql/plugin/hive/HiveWriteUtils.java

```diff
@@ -420,6 +421,11 @@ private static void checkWritable(
         if (storage.isSkewed()) {
             throw new PrestoException(NOT_SUPPORTED, format("Inserting into bucketed tables with skew is not supported. %s", tablePartitionDescription));
         }
+
+        // verify table is not ACID table
+        if (AcidUtils.isTransactionalTable(parameters)) {
+            throw new PrestoException(NOT_SUPPORTED, format("Inserting into Hive Transcational tables is not supported: %s", tableName));
```

Comment:
- no need to use `format`, since `%s` is last
- this isn't strictly related to the commit -- please extract as a separate PR